cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -271,7 +273,9 @@ def _collect(self, selection):
             view = navigate_to(self, 'DiagnosticsCollectLogs')
         last_collection = self.last_collection
         # Initiate the collection
-        view.toolbar.collect.item_select(selection)
+        view.toolbar.collect.item_select(selection, handle_alert=None)
+        if self.browser.alert_present:
+            self.browser.handle_alert(prompt='test_cfme_can_be_deleted')
```

Comment:
I see this value here and in the test, and I'm not quite sure exactly where its coming from.

Please add a comment in-line at least, and even better define it as something like `CollectLogBase.ALERT_ON_DELETE` so you can reference the constant wherever necessary for testing.